### ISSUE TITLE
greatone coldblood and thick coldblood (5) do not update when setting new quantity

### ISSUE THREAD

**Issue by watashiwalsh** [Noxde/Bloodborne-save-editor#18]
it seems all other coldblood variants update when i set their quantity to 99, but greatone and thick (5) coldblood do not. 
however, after reloading the save, it does seem to be updated. 
so it seems to be some frontend number not being updated rather than the actual save.

**Comment by Noxde** [comment]
I don't think it only happens with those items, I've had that happen with Blood stones and Madman's knowledge and it always happened on the same "slots" on some saves. 

For example if I had a Madman's knowledge that I couldn't change the amount of and I replaced it with a Coldblood, I wouldn't be able to change the amount of that specific Coldblood either so I'm not sure what causes it yet
